config/logger/base.yaml

```diff
@@ -26,6 +26,8 @@ test:
   logprobs_bootstrap_size: 10000
   # Maximum number of test data points to compute log likelihood probs.
   max_data_logprobs: 1e5
+  # Number of points tor obtain a grid to estimate the reward density
+  n_grid: 40401
```

Comment:
- Typo fixed
- I have set the default to 40000 = 200 x 200 so that it's less surprising. It's indeed coming from the Cube experiments but this can be relevant for other cases and I think it make sense to keep the default number to something we found reasonable in the Cube.
Commit: https://github.com/alexhernandezgarcia/gflownet/pull/299/commits/c9879de9ca370d93f6d21b333340c7560c98f98c